docs/sources/developers/plugins/publish-a-plugin.md

```diff
@@ -7,140 +7,39 @@ title: Publish a plugin
+In this guide, you'll learn how to share your plugin with others.
 
-- No. We cannot guarantee specific publishing dates, as plugins are immediately published after a review based on our internal prioritization.
+The best way to share your plugin with the world is to publish it in the [Grafana plugin catalog](/plugins).
+By having your plugin published on Grafana.com, more users will be able to discover your plugin.
 
-**Can I see metrics of my plugin installs, downloads or usage?**
+## Before you begin
 
-- No. We don't offer this information at the moment to plugin authors.
+Before you begin to publish a plugin, it needs to be built and packaged. 
 
-**How can I update my plugin's catalog page?**
+To build a plugin from source, Grafana creates a `dist` directory that contains the production build, or _plugin assets_, for your plugin.
 
-- The plugin's catalog page content is extracted from the plugin README file.
-  To update the plugin's catalog page, it is necessary to submit an updated plugin with the new content included in the README file.
+When the Grafana server starts, it recursively looks in the plugin directory for any directory that contains a `plugin.json` file, and then it tries to load the plugin assets in the same directory.
 
-**Can I unlist my plugin from the Grafana Plugin's Catalog in case of a bug?**
+To package a plugin, refer to [Package a plugin](package-a-plugin.md).
 
-- In the event of a bug, unlisting the plugin from the Grafana Plugin's Catalog may be possible in exceptional cases, such as security concerns.
-  However, we do not have control over the specific instances where the plugin is installed.
+### Follow our guidelines
 
-**Can I distribute my plugin somewhere else than the Grafana Catalog?**
+Get familiar with our plugin [publishing and signing criteria]({{< relref "./publishing-and-signing-criteria" >}})
 
-- The official method for distributing Grafana plugins is through the Grafana Catalog.
-  Alternative methods, such as installing private or development plugins on local Grafana instances, are available as per the guidelines provided in [this guide]({{< relref "../../administration/plugin-management#install-plugin-on-local-grafana" >}}).
+### Do this for best results
 
-**Can I still use Angular for my plugin?**
-
-- No. We will not accept any new plugin submissions written in angular.
-  Please take a look at our [angular support deprecation documentation]({{< relref "../angular_deprecation" >}}).
-
-### Publishing your plugin for the first time
+To speed up the time it takes to review your plugin:
 
-**Do plugin signatures expire?**
+- Check that your plugin is ready for review using the [plugin validator](https://github.com/grafana/plugin-validator).
+- Read our [6 tips for improving your Grafana plugin before you publish](/blog/2021/01/21/6-tips-for-improving-your-grafana-plugin-before-you-publish/).
```

Comment:
This page is quite out of date and links to deprecated tech. Suggest we remove for now and have a follow up issue to update and bring this content into the main docs

Reply:
Ok. Created https://github.com/grafana/grafana/issues/67545.